src/components/layout-components/Header/index.js

```diff
@@ -16,30 +16,33 @@ const Header = ({ headerShadow }) =>
       )}>
       <div className="container-fluid">
         <Nav pills className="mr-auto align-items-center">
-          <NavbarBrand to="/" title="Bamburgh React UI Kit with Reactstrap Free">
+          <NavbarBrand to="/" title="UC IT Expo">
             <i className="bg-transparent">
               <img
-                alt="Bamburgh React UI Kit with Reactstrap Free"
+                alt="UC Project display"
                 style={{ width: `50px` }}
                 src={projectLogo}
               />
             </i>
-            <span className="text-white pl-2">Bamburgh</span>
+            <span className="text-white pl-2">UC IT Expo</span>
           </NavbarBrand>
           <NavItem>
             <NavLink tag={Link} className="d-block mr-1" to="/">Home</NavLink>
           </NavItem>
           <NavItem>
-            <NavLink tag={Link} className="d-block mr-1" to="/blog">Blog</NavLink>
+            <NavLink tag={Link} className="d-block mr-1" to="/DevOps">DevOps</NavLink>
```

Comment:
```suggestion
            <NavLink tag={Link} className="d-block mr-1" to="/dev-ops">DevOps</NavLink>
```
Normally, URL paths are done using `snake-case` and not `PascalCase`. To change this path, you would either need to rename the file in the pages folder, or create a route in the static.config.js that maps the page to the correct URL